cmd/notary/tuf.go

```diff
@@ -860,16 +858,11 @@ const (
 // The readOnly flag indicates if the operation should be performed as an
 // anonymous read only operation. If the command entered requires write
 // permissions on the server, readOnly must be false
-func getTransport(config *viper.Viper, gun data.GUN, permission httpAccess) (http.RoundTripper, error) {
-	// Attempt to get a root CA from the config file. Nil is the host defaults.
-	rootCAFile := utils.GetPathRelativeToConfig(config, "remote_server.root_ca")
```

Comment:
this has moved to cmd/notary/main.go where all the other config parsing takes place.